frontend/src/utilities/imageUtils.ts

```diff
@@ -61,10 +63,11 @@ export const getVersion = (version?: string, prefix?: string): string => {
   if (!version) {
     return '';
   }
+
   const versionString =
-    version.startsWith('v') || version.startsWith('V') ? version.slice(1) : version;
+    version && (version.startsWith('v') || version.startsWith('V')) ? version.slice(1) : version;
 
-  return `${prefix ? prefix : ''}${versionString}`;
+  return (prefix || '') + (versionString || '');
```

Comment:
Prefer templates over concatenation:
```suggestion
  return `${prefix || ''}${versionString}`;
```